### PR TITLE
[REF] website_multi_image: moved images from binary fields to ir.attachment model

### PR DESCRIPTION
**Pull request by oscarolar**
Fixes https://github.com/Vauxoo/yoytec/issues/381

Added some improvements on look and feel on frontend

### CODE REVIEW

**Comment by moylop260** [comment]
@oscarolar 
"Y la cheyenne 'apá...?"
And the pr dummy?

**Comment by moylop260** [comment]
You have decreased 50% of coverage, because the .travis.yml is missed in your developer branch
https://github.com/vauxoo-dev/website_multi_image/tree/8.0-ir-attachment

**Comment by oscarolar** [comment]
Build: https://github.com/Vauxoo/yoytec/pull/392

File: website_multi_image/README.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Product Multi-Image
+===================
+
+This module adds multiple product images into
+a tab in product.template in a tab called
+Product Images, and allows an over-ride
+of product.image to allow for a multi-image
+carousel for the product view page.
+
+web_tree_image is recommended to install to get images preview: https://github.com/OCA/web/tree/8.0/web_tree_image
```
Comment (nhomar) [inline website_multi_image/README.rst:10]: I am agreed to install this another module in vx/yoy it is necesary in order to see the image in the list vie in the product form. Can you add such dependency in the dummy PR also please.

File: website_multi_image/static/lib/slick/config.rb [hunk 2]
```diff
@@ -7,4 +7,4 @@
 output_style = :compact
 line_comments = false
 
-preferred_syntax = :scss
+preferred_syntax = :scss
```
Comment (nhomar) [inline website_multi_image/static/lib/slick/config.rb:10]: EOF

File: website_multi_image/static/lib/slick/slick-theme.css [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+/* Icons */
+@font-face
+{
+    font-family: 'slick';
+    font-weight: normal;
+    font-style: normal;
+
+    src: url('./fonts/slick.eot');
+    src: url('./fonts/slick.eot?#iefix') format('embedded-opentype'), url('./fonts/slick.woff') format('woff'), url('./fonts/slick.ttf') format('truetype'), url('./fonts/slick.svg#slick') format('svg');
```
Comment (nhomar) [inline website_multi_image/static/lib/slick/slick-theme.css:16]: I am not sure is we should add new fonts with this module, those should be managed in the theme itself, to use new icons with this module stand alone just add some fontawsome icons should be enought don't you think?

**Comment by moylop260** [comment]
I added the missed `.travis.yml` file

**Pull request merged**